deploy/crds/push_v1alpha1_androidvariant_cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: push.aerogear.org/v1alpha1
+kind: AndroidVariant
+metadata:
+  name: example-androidvariant
+spec:
+  # Add fields here
+  variantId: 'deadbeef-badf00d'
+  name: 'example-android'
+  description: 'My super Android variant'
+  projectNumber: '123456'
+  googleKey: 'somekeyinbase64=='
```

Comment:
are `projectNumber` and `googleKey` fields actually match UPS? From UPS the UI asks for `Server Key` and `Sender Id`? Should we keep the same?

Reply:
they match the underlying data model name: https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/AndroidVariant.java#L30

don't know the UI. we can make a preference. no strong opinion here.

Reply:
renamed fields according to UI @wei-lee